Packs/Cyberpion/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "Cyberpion",
-    "description": "Cyberpion’s platform provides the breadth and depth of discovery and vulnerability assessment that security teams need to manage the threats from their far-reaching online ecosystems. Cyberpion solves the rising cybersecurity challenge of understanding the risks and vulnerabilities of your connected online assets and their artifacts. You can use this pack to push Cyberpion Action Items directly into your XSOAR instance, and also get additional information relating to those Action Items, and much more",
+    "description": "Cyberpion\u2019s platform provides the breadth and depth of discovery and vulnerability assessment that security teams need to manage the threats from their far-reaching online ecosystems. Cyberpion solves the rising cybersecurity challenge of understanding the risks and vulnerabilities of your connected online assets and their artifacts. You can use this pack to push Cyberpion Action Items directly into your XSOAR instance, and also get additional information relating to those Action Items, and much more",
```

Comment:
Is this change intentional?

Reply:
You are right, that's a mistake. Fixed it. Thanks!